android/pytorch_android/style_transfer_cyclegan/README.md

```diff
@@ -0,0 +1,16 @@
+# DJL Android Demo
+
+# Introduction
+In this example, you will learn how to do style transfer with pre-trained CycleGAN models. You will transform regular images into a painting by a famous artist. The supported artists are Cezanne, Monet, Ukiyoe and Vangogh.
+
+## Style Transfer
+
+Cezanne          |  Monet            | Ukiyoe        |  Vangogh       
+:--------------:|:-----------------:|:----------------:|:--------------:
+![](img/cezanne.jpg)        |  ![](img/monet.jpg)         | ![](img/ukiyoe.jpg)        |  ![](img/vangogh.jpg)         
```

Comment:
I have uploaded image to: https://resources.djl.ai/images/cyclegan folder. Please update the link